src/main/java/woowa/bossdog/subway/repository/FavoriteRepository.java

```diff
@@ -0,0 +1,17 @@
+package woowa.bossdog.subway.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import woowa.bossdog.subway.domain.Favorite;
+
+import java.util.List;
+
+public interface FavoriteRepository extends JpaRepository<Favorite, Long> {
+
+    @Query("select f from Favorite f where f.member.id = :memberId")
+    List<Favorite> findAllByMemberId(@Param("memberId") Long memberId);
```

Comment:
메서드 이름으로 쿼리를 생성하면 어떨까요?

```suggestion
    List<Favorite> findAllByMemberId(Long memberId);
```

Reply:
반영했습니다.!